tools/github_workflows/workflows_template.libsonnet

```diff
@@ -2,60 +2,76 @@
   local platforms = [
     {
       name: 'linux_amd64',
-      buildAndTestCommand: 'test --test_output=errors',
       buildJustBinaries: false,
```

Comment:
Is `buildJustBinaries: true` still used at this point? If not, probably good to remove this flag entirely?